test/fetch-wrapper.js

```diff
@@ -0,0 +1,19 @@
+const { fetch } = require('../src/fetch-wrapper');
+const assert = require('assert');
+
+describe('fetchWrapper', () => {
+  it('exports fetch(), a wrapper around node-fetch', () => {
+      assert.equal(typeof fetch, 'function');
+  });
+  it('emits a custom APIError on failure', () => {
+      // an empty request will fail
+      fetch()
+      .then(error => {
+          assert(error.response);
+          assert(error.message);
+          assert(error.status);
+          assert(error.toString);
+      })
+      .catch(() => {});
+  });
```

Comment:
This test should be removed, right?

Reply:
Yes good catch. Also just realized, we should probably have tests here that ensure the wrapper module chooses the correct `fetch`.